maintainers-guide.md

```diff
@@ -0,0 +1,210 @@
+
+## My team owns a label. What should I do?
+
+Subteams need to triage all issues in the [labels they own](), preferable on a weekly basis.
+
+### Issues
+
+1. Filter the list of issues by your team label **and** the `untriaged` label.
+1. Review the issue.
+1. Identify a priority level and assign the label.
+  1. The issue may have already been prioritized by the DevEx subteam if it's a P0. Re-prioritize if needed.
+1. Remove the `untriaged` label.
+
+
+Note that you need to be in the [bazelbuild
+organization](https://github.com/bazelbuild) to be able to add or remove labels.
+
+### Pull Requests
+
+1. Filter the list of pull requests by your team label.
+1. Review open pull requests.
+  1. **Optional**: If you are assigned for the review but is not the right fit
+  for it, re-assign the appropriate reviewer to perform a code review.
+1. Work with the pull request creator to complete a code review.
+1. Approve the PR.
+1. Ensure that all tests pass.
+1. Import the patch to the internal version control system and run the internal
+   presubmits.
+1. Submit the internal patch. If the patch submits and exports successfully, the
+   PR will be closed automatically by GitHub.
+
+## Priority
+
+The following definitions for priority will be used by the maintainers to triage
+issues.
+
+* **P0** - Major broken functionality that causes a Bazel release (minus release
+  candidates) to be unusable, or a downed service that severely impacts
+  development of the Bazel project. This includes regressions introduced in a
+  new release that blocks a significant number of users, or an incompatible
+  breaking change that was not compliant to the [Breaking
+  Change](https://docs.google.com/document/d/1q5GGRxKrF_mnwtaPKI487P8OdDRh2nN7jX6U-FXnHL0/edit?pli=1#heading=h.ceof6vpkb3ik)
+  policy.
+* **P1** - Critical feature request or bug fix which should be addressed in the
+  next release, or a serious issue that impacts many users. Typically does not
+  require immediate action. High demand, planned in the roadmap for the current
+  quarter.
+* **P2** - Important feature or defect that is tied to roadmaps or upcoming
+  releases. Moderate live issue in a released Bazel version that is inconvenient
+  for a user that needs to be addressed in a upcoming release.
+* **P3** - Desirable enhancement or minor bug fix that will benefit a user. Not
+  prioritized into Bazel roadmaps or any imminent release.
+* **P4** - Low priority defect or feature request that is unlikely to get
+  closed. Can also be kept open for a potential re-prioritization if more users
+  are impacted.
+
+## Team labels
+
+| Label                | Description                                                           | Link                                                                           |
+|----------------------|-----------------------------------------------------------------------|--------------------------------------------------------------------------------|
```

Comment:
It may be easier to maintain as a list?
I think it would be more flexible also, if we want to add a description or more information

e.g.

* [`team-Android`](https://github.com/bazelbuild/bazel/labels/team-Android): Issues for Android team